packages/ui/src/views/labels/labels-list-page.tsx

```diff
@@ -61,62 +62,55 @@ export const LabelsListPage: FC<LabelsListPageProps> = ({
 
   return (
     <SandboxLayout.Main>
-      <SandboxLayout.Content className="px-0">
-        <h1 className="text-2xl font-medium text-foreground-1">{t('views:labelData.title', 'Labels')}</h1>
-        <Spacer size={6} />
+      <SandboxLayout.Content className={className}>
+        <h1 className="mb-6 text-2xl font-medium text-foreground-1">{t('views:labelData.title', 'Labels')}</h1>
+
         {isRepository && (
           <Option
             className="mb-[18px]"
             control={
-              <Checkbox
-                checked={getParentScopeLabels}
-                onCheckedChange={setGetParentScopeLabels}
-                id="show-parent-labels"
-              />
+              <Checkbox checked={getParentScopeLabels} onCheckedChange={setGetParentScopeLabels} id="parent-labels" />
             }
-            id="show-parent-labels"
+            id="parent-labels"
             label={t('views:labelData.showParentLabels', 'Show labels from parent scopes')}
           />
         )}
 
         {(!!spaceLabels.length || (!spaceLabels.length && isDirtyList)) && (
```

Comment:
Am I crazy for thinking this could just be...
```suggestion
        {(!!spaceLabels.length || isDirtyList) && (
```

🤔